Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -137,7 +137,7 @@ public class PBXProjGenerator {
 
         // sort children
         let children: [GroupChild] = group.children
-            .flatMap { proj.objects.groups[$0] as GroupChild? }
+            .flatMap { proj.objects.groups[$0] ?? proj.objects.fileReferences[$0] }
```

Comment:
needs `.variantGroup` as well

Reply:
In fact I just noticed we already have an extension on PBXProj called `getGroupChild(reference)` that you can use

Reply:
Much better!